build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -425,6 +425,19 @@ message Action {
   bool do_not_cache = 7;
 
   reserved 8; // Used for field moved to [Command][build.bazel.remote.execution.v2.Command].
+
+  // An optional additional salt value used to place this `Action` into a
+  // separate cache universe or "epoch" from other instances having the
```

Comment:
'cache universe or "epoch"' I'd probably go with the more generic "cache namespace" here.

Reply:
Updated.

Reply:
updated.